staff/carme-casserras/cv/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>CV CARMECASS</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="css/style.css" rel="stylesheet">
+    </head>
+    <body>
+    <header>
+        <h2>CARME CASSERRAS VENDRELL</h2>
+            <section>
+                <p>DIRECCIÓN ACTUAL</p>
+                <p>Sant Jeroni 10, bajos 1</p>
+                <p>08340 VILASSAR DE MAR (BARCELONA)</p>
+                <p>Telf. 610 70 20 35</p>              
+            </section>
+            <section>
+                <nav>
+                    <ul>
+                        <li>
+                            <a href="carmecass@gmail.com">carmecass@gmail.com</a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/messaging/compose/?after=profile.view&afterParam=carme-casserras-736b06b&body=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fcarme-casserras-736b06b&lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base%3BB2x30DFsSqG2VUD%2B3br0KA%3D%3D&licu=urn%3Ali%3Acontrol%3Ad_flagship3_profile_view_base-message">Linkedin</a>
+                        </li>
+                    </ul>
+                </nav>
+            </section>
+            <aside>
+                <img src="https://ca.slack-edge.com/T0SJKHBFZ-UEETBFA2W-b9014c205756-72">
```

Comment:
@carmecass "alt" attribute is highly recommended in all img tags. Please check in the following link: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img .